Bio/UniGene/__init__.py

```diff
@@ -324,8 +328,11 @@ def _read(handle):
             record.sts.append(sts)
         elif tag == "//":
             if len(record.sequence) != scount:
-                raise ValueError("The number of sequences specified in the record"
-                                 " (%d) does not agree with the number of sequences found (%d)" % (scount, len(record.sequence)))
+                raise ValueError(
+                    "The number of sequences specified in the record"
```

Comment:
We don't have any automated check for this, but I think we usually put the trailing space on end of the previous line, rather than a leading space on the continuation line.

Reply:
Ok, done.